lane.py

```diff
@@ -675,7 +675,7 @@ def modify_database_query(self, _db, qu):
         available_now = or_(
             LicensePool.open_access == True,
             LicensePool.self_hosted == True,
-            LicensePool.unlimited_access == True,
+            LicensePool.unlimited_access,
```

Comment:
This doesn't make a difference, but it's consistent with what we do later in this method, and it makes it clear that unlimited_access is a synthetic property rather than a boolean stored in the database.

Reply:
What do you mean by synthetic property? Is it like having _any_ value instead of explicitly being a boolean?

Reply:
No, it should always be a boolean. I meant that `LicensePool.unlimited_access` is defined in Python through `@property`, not as a real database field. I'd never seen a Python property used in a SQLAlchemy query before but I verified this works when reviewing Viacheslav's original code.